pkg/ingress/controller/controller.go

```diff
@@ -95,6 +95,10 @@ const (
 	// Default to true.
 	IngressAnnotationInternal = "octavia.ingress.kubernetes.io/internal"
 
+	// IngressAnnotationSourceRangesKey is the key of the annotation on an ingress to set allowed IP ranges on their LoadBalancers.
+	// It should be a comma-separated list of CIDRs.
+	IngressAnnotationSourceRangesKey = "ingress.kubernetes.io/whitelist-source-range"
```

Comment:
maybe better name for this is `octavia.ingress.kubernetes.io/whitelist-source-range`? because its octavia ingress controller specific setting, not whole kubernetes

Reply:
you're totally right, as others annotations are also prefixed by `octavia.ingress.kubernetes.io`.